dev-qt/qtprotobuf/qtprotobuf-0.1.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="gRPC and Protobuf generator and bindings for Qt framework"
+HOMEPAGE="https://semlanik.github.io/qtprotobuf/"
+SRC_URI="https://github.com/semlanik/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
```

Comment:
You probably have not tested this ebuild on prefix architectures (`~amd64-linux ~x86-linux`).
See comment about `CMAKE_INSTALL_PREFIX` below.